### PR TITLE
Php scripts 3

### PR DESCRIPTION
**Pull request by JackAmos**
I re-based it onto master so there shouldn't be any conflicts. This adds post and reply editing and deletion.

### CODE REVIEW

**Comment by JackAmos** [comment]
Fixes https://github.com/tzmartin5/CS4420/issues/27 and https://github.com/tzmartin5/CS4420/issues/28

**Comment by JackAmos** [comment]
Fixes https://github.com/tzmartin5/CS4420/issues/30

File: php/delete_post.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?php
+
+session_start();
+ini_set('display_startup_errors', 1);
+ini_set('display_errors', 1);
+error_reporting(-1);
+
+if(isset($_SESSION['group7_username'])) {
```
Comment (gurbaflurb) [inline php/delete_post.php]: Do we already have this implemented? I thought I saw that we could already delete an entire post already.

File: php/register.php [hunk 2]
```diff
@@ -86,6 +86,19 @@
...
 					mysqli_stmt_execute($stmt);
+
+					//email contents
+					$to = $email;
+					$subject = "signup for Mothers Against AI";
+					$message = "Thank you for signing up! Please enjoy our website!";
+
+					$headers = "From: www.mothersagainstai.com <@>\r\n";//email sent from
+					$headers .= "Reply To: www.mothersagainstai.com <@>\r\n";//email to reply to
+					$headers .= "Content-type: text/html\r\n";
```
Comment (gurbaflurb) [inline php/register.php]: Should these be actual email address instead of a url?

File: php/delete_post.php (hunk 1 above)
Comment (JackAmos) [reply php/delete_post.php]: I haven't seen any scripts that would, I made this script a while ago, but I don't think anyone else submitted a script to delete

File: php/register.php (hunk 2 above)
Comment (JackAmos) [reply php/register.php]: Unfortunately we cannot as we don't have an email address, since they didn't have us set up a mail server

**Comment by JackAmos** [comment]
I found del_post.php, it doesn't delete the attached replys tho

**Pull request merged**